### PR TITLE
Add Pragli

### PR DESCRIPTION
**Pull request by connorads**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/pragli.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "pragli" do
+  version :latest
+  sha256 :no_check
+
+  url "https://storage.googleapis.com/always-on-cdf01.appspot.com/dist/Pragli.dmg",
+      verified: "storage.googleapis.com/always-on-cdf01.appspot.com"
```
Comment (connorads) [inline Casks/pragli.rb:6]: This is the official url for the mac app as per the [Pragli download page](https://pragli.com/blog/desktop-client-downloads/).

image

It looks like it's a Google/Firebase thing. The main website seems to be hosted on Firebase at a similar url with `always-on-cdf01` in it: https://always-on-cdf01.firebaseapp.com/

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @connorads.

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by miccal** [comment]
Thank you @connorads.

**Review by miccal — APPROVED** [review]

**Comment by connorads** [comment]
Thank you @miccal!

**Pull request merged**